<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Setup</class>
 <widget class="QDialog" name="Dialog_Setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../obpm.qrc">
    <normaloff>:/png/png/setup.png</normaloff>:/png/png/setup.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="icon">
       <iconset resource="../obpm.qrc">
        <normaloff>:/png/png/general.png</normaloff>:/png/png/general.png</iconset>
      </attribute>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Alias</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../obpm.qrc">:/png/png/user1.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_user1">
            <property name="toolTip">
             <string>enter name for user 1</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../obpm.qrc">:/png/png/user2.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_user2">
            <property name="toolTip">
             <string>enter name for user 2</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Database</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../obpm.qrc">:/png/png/database.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_db">
            <property name="toolTip">
             <string>current database location</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_db">
            <property name="toolTip">
             <string>choose database location</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../obpm.qrc">
              <normaloff>:/png/png/folder.png</normaloff>:/png/png/folder.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Update</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkBox_update">
            <property name="toolTip">
             <string>online update on/off</string>
            </property>
            <property name="text">
             <string>check for Online Updates when Program starts</string>
            </property>
            <property name="icon">
             <iconset resource="../obpm.qrc">
              <normaloff>:/png/png/work.png</normaloff>:/png/png/work.png</iconset>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../obpm.qrc">
        <normaloff>:/png/png/appearance.png</normaloff>:/png/png/appearance.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_1">
         <property name="title">
          <string>Defaults</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="3">
             <widget class="QLabel" name="label_sys">
              <property name="text">
               <string notr="true">135</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLabel" name="label_bpm">
              <property name="text">
               <string notr="true">80</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../obpm.qrc">:/png/png/bpm.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QSlider" name="horizontalSlider_sys">
              <property name="toolTip">
               <string>modify medical specifications</string>
              </property>
              <property name="minimum">
               <number>115</number>
              </property>
              <property name="maximum">
               <number>155</number>
              </property>
              <property name="pageStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>135</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSlider" name="horizontalSlider_bpm">
              <property name="toolTip">
               <string>modify medical specifications</string>
              </property>
              <property name="minimum">
               <number>60</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="pageStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>80</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../obpm.qrc">:/png/png/dia.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSlider" name="horizontalSlider_dia">
              <property name="toolTip">
               <string>modify medical specifications</string>
              </property>
              <property name="minimum">
               <number>65</number>
              </property>
              <property name="maximum">
               <number>105</number>
              </property>
              <property name="pageStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>85</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_dia">
              <property name="text">
               <string notr="true">85</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_1">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../obpm.qrc">:/png/png/sys.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label">
              <property name="text">
               <string notr="true">SYS</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string notr="true">DIA</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string notr="true">BPM</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Printout</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox_psd">
            <property name="toolTip">
             <string>single diagram on/off</string>
            </property>
            <property name="text">
             <string>Blood Pressure &amp;&amp; Heart Rate on single Page</string>
            </property>
            <property name="icon">
             <iconset resource="../obpm.qrc">
              <normaloff>:/png/png/pages.png</normaloff>:/png/png/pages.png</iconset>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_pot">
            <property name="toolTip">
             <string>table overview on/off</string>
            </property>
            <property name="text">
             <string>include Table Overview with Measurements</string>
            </property>
            <property name="icon">
             <iconset resource="../obpm.qrc">
              <normaloff>:/png/png/table.png</normaloff>:/png/png/table.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_ihb">
            <property name="toolTip">
             <string>Irregular beat  on/off</string>
            </property>
            <property name="text">
             <string>Include irregular heart beat in graph</string>
            </property>
            <property name="icon">
             <iconset resource="../obpm.qrc">
              <normaloff>:/png/png/heart1.png</normaloff>:/png/png/heart1.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="icon">
       <iconset resource="../obpm.qrc">
        <normaloff>:/png/png/work.png</normaloff>:/png/png/work.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Delete User Data</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Take care!! &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;You are about to delete the &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;database table for a dedicated user.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;All user info will be lost !!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="deluser1">
           <property name="text">
            <string>Delete User 1 data </string>
           </property>
           <property name="icon">
            <iconset resource="../obpm.qrc">
             <normaloff>:/png/png/user1.png</normaloff>:/png/png/user1.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="deluser2">
           <property name="text">
            <string>Delete User 2 data</string>
           </property>
           <property name="icon">
            <iconset resource="../obpm.qrc">
             <normaloff>:/png/png/user2.png</normaloff>:/png/png/user2.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="toolTip">
        <string>reset settings to defaults</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="icon">
        <iconset resource="../obpm.qrc">
         <normaloff>:/png/png/reset.png</normaloff>:/png/png/reset.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="toolTip">
        <string>save settings</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../obpm.qrc">
         <normaloff>:/png/png/save.png</normaloff>:/png/png/save.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../obpm.qrc"/>
 </resources>
 <connections/>
</ui>
